DESCRIPTION

```diff
@@ -41,13 +41,16 @@ Suggests:
     knitr,
     Lahman,
     nycflights13,
+    odbc (>= 1.2.2.9000),
     RMariaDB (>= 1.0.2),
     rmarkdown,
     RPostgres (>= 1.1.3),
     RSQLite (>= 2.1.0),
     testthat (>= 2.0.0)
 VignetteBuilder: 
     knitr
+Remotes: 
```

Comment:
Can you not use remotes, and skip if the version isn't high enough? That way I don't have to worry about updating it when I release to CRAN?